extensions/EXT/SPV_EXT_shader_atomic_float16_add.asciidoc

```diff
@@ -98,7 +98,7 @@ Add the *AtomicFloat16AddEXT* capability to the *OpAtomicFAddEXT* instruction ad
 
 1+|Capability: +
 *AtomicFloat32AddEXT* *AtomicFloat64AddEXT* *AtomicFloat16AddEXT*
-| 7 | 6035 | _<id> Result type_ | _Result <id>_ | _<id>_ Pointer  | _Scope <id> Memory_ | Memory Semantics <id> Semantics | <id> Value |
+| 7 | 6035 | _<id> Result type_ | _Result <id>_ | _<id>_ Pointer  | _Scope <id> Memory_ | Memory Semantics <id> Semantics | <id> Value
```

Comment:
asciidoctor: ERROR: SPV_EXT_shader_atomic_float16_add.asciidoc: line 101: dropping cells from incomplete row detected end of table

Reply:
Oh no, that's what happens without this PR.. I wonder if the new CI is building the wrong branch..